### DISCUSSION TITLE
new user questions

### DISCUSSION THREAD

**Discussion by ralfswan** [sumatrapdfreader/sumatrapdf#4675] [Documentation]
1. What are the file names that appear at the top of the screen when I open a file with Sumatrapdf? 
2. I am having difficulty entering the from and thru page numbers into the "Custom Range" on the Print screen
![SumatraPDF 1](https://github.com/user-attachments/assets/173c3278-6a4a-4693-844c-bafe33f02776)
![SumatraPDF 2](https://github.com/user-attachments/assets/2ebf890d-09e5-4e0d-9a28-4a093904425e)
.

**Comment by GitHubRulesOK** [comment]
1) The filenames are the ones restored **from last session** and should be visible (certainly in current pre-release) as a tooltip
![image](https://github.com/user-attachments/assets/58f3b12b-3ffc-4162-a951-c79bc007a3bb)

2) Microsoft printing in recent versions such as 11 use a new dialog and reportedly seem to have problems with supporting the range entries. But I am currently using older 10 so not seen that personally!
